src/software/ai/hl/stp/tactic/crease_defender/crease_defender_tactic.cpp

```diff
@@ -45,8 +48,16 @@ void CreaseDefenderTactic::updatePrimitive(const TacticUpdate &tactic_update,
     if (reset_fsm)
     {
         fsm_map[tactic_update.robot.id()] = std::make_unique<FSM<CreaseDefenderFSM>>(
-            CreaseDefenderFSM(robot_navigation_obstacle_config));
+            CreaseDefenderFSM(ai_config.robot_navigation_obstacle_config()),
+            DribbleFSM(ai_config.dribble_tactic_config()));
     }
+    DribbleFSM::ControlParams dribble_control_params{
+            .dribble_destination = Point(),
+            .final_dribble_orientation = Angle(),
+            .allow_excessive_dribbling = false
+    };
     fsm_map.at(tactic_update.robot.id())
         ->process_event(CreaseDefenderFSM::Update(control_params, tactic_update));
+    fsm_map.at(tactic_update.robot.id())
+        ->process_event(DribbleFSM::Update(dribble_control_params, tactic_update));
```

Comment:
do we need this? I'm not 100% sure if this is necessary

Reply:
No idea why, but weirdly yes